microsoft/testsuites/network/sriov.py

```diff
@@ -501,21 +502,24 @@ def verify_sriov_reload_modules(self, environment: Environment) -> None:
             ),
         ),
     )
-    def verify_sriov_ethtool_offload_setting(self, environment: Environment) -> None:
+    def verify_sriov_ethtool_offload_setting(  # noqa: C901
```

Comment:
Split some functions, instead of add the noqa.

Reply:
updated